shippable.yml

```
@@ -34,6 +35,7 @@ matrix:
     - env: T=devel/linux/ubuntu1604/1
     - env: T=devel/linux/ubuntu1804/1
 
+    - env: T=devel/cloud/2.6/1
     - env: T=devel/cloud/2.7/1
     - env: T=devel/cloud/3.6/1
```

While you're at it, why not also add:
```suggestion
    - env: T=devel/cloud/3.5/1
    - env: T=devel/cloud/3.6/1
    - env: T=devel/cloud/3.7/1
    - env: T=devel/cloud/3.8/1
    - env: T=devel/cloud/3.9/1
```

---

Sure, I thought they were not in for some reason.

---

They were mostly not in because that's what ansible/ansible ran.